core/src/container.cpp

```diff
@@ -183,10 +183,8 @@ void ContainerBasePrivate::setStatus(const InterfaceState* s, InterfaceState::St
 	for (const SolutionBase* successor : trajectories<dir>(*s))
 		setStatus<dir>(state<dir>(*successor), status);
 }
-template void ContainerBasePrivate::setStatus<Interface::FORWARD>(const InterfaceState* s,
-                                                                  InterfaceState::Status status);
-template void ContainerBasePrivate::setStatus<Interface::BACKWARD>(const InterfaceState* s,
-                                                                   InterfaceState::Status status);
+template void ContainerBasePrivate::setStatus<Interface::FORWARD>(const InterfaceState*, InterfaceState::Status);
+template void ContainerBasePrivate::setStatus<Interface::BACKWARD>(const InterfaceState*, InterfaceState::Status);
```

Comment:
Some part of my toolchain keeps changing these lines back, sorry.
I just tried to find the offender though and everything behaves well, now that I look for the source... :confused: